### PR TITLE
LPS-125611 Check if value is null or undefined using the != operator

### PR DESCRIPTION
**Pull request by carolmariaabb**
Related Issue: https://issues.liferay.com/browse/LPS-125611

### CODE REVIEW

**Comment by liferay-forms** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
Hi, you don’t need to run the CI commands anymore, I’ll do it for you and let you know if any issue happens.

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
Hey @carolmariaabb, can you please check if these failures are related to your changes? Thanks

**Comment by carolmariaabb** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
Hey @carolmariaabb, can you please check if these failures are related to your changes? Thanks

**Comment by carolmariaabb** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
ci:test:relevant

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/test/java/com/liferay/dynamic/data/mapping/internal/report/NumericDDMFormFieldTypeReportProcessorTest.java [hunk 1]
```diff
@@ -80,12 +81,24 @@ public void testFormatBigDecimal() {
 				new BigDecimal("12.0000000010000")));
 	}
 
+	@Test
+	public void testGetValueBigDecimal() {
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/test/java/com/liferay/dynamic/data/mapping/internal/report/NumericDDMFormFieldTypeReportProcessorTest.java:85]: Lovely job, @carolmariaabb! 😄 

leonardo di caprio

**Review by natocesarrego — APPROVED** [review]

**Comment by natocesarrego** [comment]
ci:forward

**Pull request closed without merging**